frontend/packages/knative-plugin/src/utils/__tests__/knative-serving-data.ts

```diff
@@ -416,3 +416,184 @@ export const getEventSourceDeploymentData = (EventSourceModel: K8sKind): K8sReso
     },
   };
 };
+
+export const Kafkas: K8sResourceKind[] = [
+  {
+    apiVersion: 'kafka.strimzi.io/v1beta1',
+    kind: 'Kafka',
+    metadata: {
+      creationTimestamp: '2020-07-06T16:43:44Z',
+      generation: 1,
+      managedFields: [
+        {
+          apiVersion: 'kafka.strimzi.io/v1beta1',
+          fieldsType: 'FieldsV1',
+          fieldsV1: {
+            'f:status': {
+              'f:conditions': {},
+              'f:listeners': {},
+              'f:observedGeneration': {},
+            },
+          },
+          manager: 'okhttp',
+          operation: 'Update',
+          time: '2020-07-06T16:45:13Z',
+        },
+      ],
+      name: 'my-cluster',
+      namespace: 'div',
+      resourceVersion: '142204',
+      selfLink: '/apis/kafka.strimzi.io/v1beta1/namespaces/div/kafkas/my-cluster',
+      uid: '56c871cf-e649-4c7e-8ac6-8fafc933c96f',
+    },
+    spec: {
+      kafka: {
+        config: {
+          'log.message.format.version': '2.5',
+          'offsets.topic.replication.factor': 3,
+          'transaction.state.log.min.isr': 2,
+          'transaction.state.log.replication.factor': 3,
+        },
+        listeners: {
+          plain: {},
+          tls: {},
+        },
+        replicas: 3,
+        storage: {
+          type: 'ephemeral',
+        },
+        version: '2.5.0',
+      },
+      zookeeper: {
+        replicas: 3,
+        storage: {
+          type: 'ephemeral',
+        },
+      },
+    },
+    status: {
+      conditions: [
+        {
+          lastTransitionTime: '2020-07-06T16:45:13+0000',
+          status: 'True',
+          type: 'Ready',
+        },
+      ],
+      listeners: [
+        {
+          addresses: [
+            {
+              host: 'my-cluster-kafka-bootstrap.div.svc',
+              port: 9092,
+            },
+          ],
+          bootstrapServers: 'my-cluster-kafka-bootstrap.div.svc:9092',
+          type: 'plain',
+        },
+        {
+          addresses: [
+            {
+              host: 'my-cluster-kafka-bootstrap.div.svc',
+              port: 9093,
+            },
+          ],
+          bootstrapServers: 'my-cluster-kafka-bootstrap.div.svc:9093',
+          certificates: [
```

Comment:
do we need `certificates` and `managedFields` in mockData

Reply:
Not as such but just wanted to keep the resource same as what is returned after creating.